src/spi/cppthreads/CppThreadsTimer.cpp

```diff
@@ -35,13 +37,8 @@ bool CppThreadsTimer::start(uint16_t timeout, std::function<void (ITimer* trigge
 	}
 	else {
 		this->started = true;
-		this->waitingThread = std::thread([=]() {
-			std::unique_lock<std::mutex> lock(this->cv_m);
-			this->cv.wait_for(lock, std::chrono::milliseconds(timeout), [this]{return !this->started;});
-			if (this->started) {
-				callBackFunction(this);
-			}
-		});
+    this->callback = callBackFunction;
```

Comment:
(minor) Indent

Reply:
Done,
The indentations are not the same inside the files